pkg/ansible/watches/watches.go

```diff
@@ -54,6 +54,7 @@ type Watch struct {
 	WatchDependentResources     bool                      `yaml:"watchDependentResources"`
 	WatchClusterScopedResources bool                      `yaml:"watchClusterScopedResources"`
 	Selector                    metav1.LabelSelector      `yaml:"selector"`
+	SnakeCaseParameters         bool                      `yaml:"snakeCaseParameters"`
```

Comment:
The issue faced in the lint is `maligned`. To solve it you can just keep the bools at the ends. Move the Selector before of them.  

```

To know more about it see: https://medium.com/@sebassegros/golang-dealing-with-maligned-structs-9b77bacf4b97
Running golangci-lint
pkg/ansible/watches/watches.go:44:12: struct of size 200 bytes could be of size 192 bytes (maligned)
type Watch struct {
           ^
Makefile:89: recipe for target 'lint' failed
make: *** [lint] Error 1
The command "make test-sanity" exited with 2.
cache.2
store build cache
```

Reply:
I'm speechless at this lint check.

Reply:
Hi @tengqm, 

For further information see: https://medium.com/@sebassegros/golang-dealing-with-maligned-structs-9b77bacf4b97

Reply:
@camilamacedo86 Thanks for the pointer. I didn't realize that Go is so hypercritical and nitpicking ... and I'm not sure if it is always a good thing. But anyway.

Reply:
Just to clarifies, the `maligned` is a lint check. It checks the usage of memory space of a structure and alert when we are using more spaces than required. Why it would be a bad thing at all? 

However, in the cases that we decide to prioritize the code understatement instead of it, we can just say for the lint ignore the specific check in the specific place by using `// nolint:maligned`

Reply:
If it is always the right thing to do, the compiler should do this byte alignment transparently rather than forcing the developer to think about these details. There are other cases I want to order my fields for different purposes, e.g. semantic grouping, deriving Web UI for an input struct etc.